cms/utils/conf.py

```diff
@@ -254,6 +255,12 @@ def get_unihandecode_host():
         return host + '/'
 
 
+def get_placeholder_config():
+    name = 'PLACEHOLDER_CONF'
+    placeholder_conf = getattr(settings, 'CMS_%s' % name, DEFAULTS[name])
+    return OrderedDict(placeholder_conf)
```

Comment:
If `CMS_PLACEHOLDER_CONF` is not an `OrderedDict` when defined in the settings, casting it to an `OrderedDict` afterwards will not keep the ordering.

```python
>>> d = {'1': 1, '2': 2, '3': 3}
>>> from collections import OrderedDict
>>> OrderedDict(d)
OrderedDict([('1', 1), ('3', 3), ('2', 2)])
>>> d
{'1': 1, '3': 3, '2': 2}
```